packages/iframe-coordinator/src/messages/Click.ts

```diff
@@ -0,0 +1,18 @@
+import { constant, Decoder, object } from 'decoders';
+import { labeledDecoder, LabeledMsg } from './LabeledMsg';
+
+/**
+ * A message used to notify the host that a click (mousedown followed by mouseup)
+ * occurred in the client application.
+ */
+export interface LabeledClick extends LabeledMsg<'clickFired', {}> {
+  /** Message identifier */
+  msgType: 'clickFired';
```

Comment:
Is it just me or is this name a bit ambiguous about if some active code is "firing" a click or if the user just clicked on something?  The terminology around events being fired/triggered/emitted is frustratingly varied. Would "appClicked" or "userClick" be clearer? It might just be me.

Reply:
I opted for consistency with the existing registeredKeyFired for KeyDown. I am happy to go with whatever name makes the most sense to everyone.